soliket/mflike/MFLike.yaml

```diff
@@ -20,10 +23,10 @@ defaults:
   polarizations: ['TT', 'TE', 'ET', 'EE']
   # Scale cuts (in ell) for each spectrum
   scales:
-    TT: [2, 5000]
-    TE: [2, 5000]
-    ET: [2, 5000]
-    EE: [2, 5000]
+    TT: [50, 5000]
```

Comment:
is 5000 the correct lmx for data in yaml?

Reply:
not sure, I just used what was left there

Reply:
ok! This might not be relevant for the current release, but maybe good to check with PS whether this is the correct lmax for data cut